### DISCUSSION TITLE
Using doks theme with hugo serve blank page

### DISCUSSION THREAD

**Discussion by wphromma** [thuliteio/doks#603] [Q&A] [answered]
Hi, I am new to Hugo, and I just follow the get started guide from [https://gohugo.io/getting-started/quick-start/](https://gohugo.io/getting-started/quick-start) 

I can get the theme working with the theme `ananke` from the guide. However, I want to try `doks` theme, and when I replace `ananke` theme with `doks` theme, it's serving the blank page with some warning messages.

I wonder if there is something I'm missing? how do you get it to work with Hugo?

and FYI, I tried to use `doks` alone from this guide [https://getdoks.org/docs/prologue/quick-start/](https://getdoks.org/docs/prologue/quick-start) and it's working properly but not with Hugo.

Kidnly advise how can I solve this issue.

Thanks

<img width="1678" alt="Screen Shot 2022-01-08 at 3 52 20 AM" src="https://user-images.githubusercontent.com/72568309/148606346-aafa19b0-937b-4359-871e-311de2ce21a6.png">
<img width="1667" alt="Screen Shot 2022-01-08 at 3 53 06 AM" src="https://user-images.githubusercontent.com/72568309/148606366-66d5f4d4-8014-476f-9064-93e02c937c7a.png">

**Comment by h-enk** [comment, marked as answer]
Doks is not — on purpose — a standard Hugo theme. 

Unfortunately, Hugo does not provide a (solid) way to centralize dependency management. Doks uses npm to centralize dependency management — making it easy to update resources, build tooling, plugins, and build scripts. When you put Doks in the theme folder, it won't work, because Hugo does not support this — in the JavaScript world standard — npm setup.

There are a number of reasons why I don't like setting up a theme as a Git submodule or a Hugo module. If you're interested /would like to know why, I wrote a little piece on that, see [Adding a theme](https://henkverlinde.com/managing-dependencies/#adding-a-theme), part of the [Using npm with Hugo](https://henkverlinde.com/introduction-to-npm/) series.

So, the [Quick Start](https://getdoks.org/docs/prologue/quick-start/) is the way to get started with Doks!

**Reply by wphromma** [reply]
Are you saying that I should customize the theme not from the Hugo on top of it but straight to the theme itself?

**Reply by h-enk** [reply]
Yes. You could see Doks as a standalone theme, where Hugo is not a requirement, but a dependency!
